test/csuite/schema_abort/main.c

```diff
@@ -541,18 +554,27 @@ thread_ckpt_run(void *arg)
      * Keep a separate file with the records we wrote for checking.
      */
     (void)unlink(ckpt_file);
-    memset(buf, 0, sizeof(buf));
+    memset(ckpt_flush_config, 0, sizeof(ckpt_flush_config));
+    memset(ckpt_config, 0, sizeof(ckpt_config));
     testutil_check(td->conn->open_session(td->conn, NULL, NULL, &session));
     first_ckpt = true;
     ts = 0;
+    flush_tier = false;
+
+    testutil_check(__wt_snprintf(ckpt_config, sizeof(ckpt_config), "use_timestamp=true"));
+
+    testutil_check(__wt_snprintf(
+      ckpt_flush_config, sizeof(ckpt_flush_config), "flush_tier=(enabled,force),%s", ckpt_config));
+
+    set_flush_tier_delay(&rnd);
+
     /*
      * Keep writing checkpoints until killed by parent.
      */
     __wt_epoch(NULL, &start);
     for (i = 1;; ++i) {
         sleep_time = __wt_random(&rnd) % MAX_CKPT_INVL;
-        sleep(sleep_time);
-        flush = ((__wt_random(&rnd) % FLUSH_INVL) == 0) ? true : false;
+        testutil_tiered_sleep(opts, session, sleep_time, &flush_tier);
```

Comment:
Didn't this need a corresponding change to `testutil_tiered_sleep` to work?  Something along these lines:
```
index 47bab6adf..3666b0d41 100644
--- a/test/utility/tiered.c
+++ b/test/utility/tiered.c
@@ -65,6 +65,11 @@ testutil_tiered_sleep(TEST_OPTS *opts, WT_SESSION *session, uint32_t seconds, bo
     now = testutil_time_us(session);
     wake_time = now + 1000000 * seconds;
     do_flush = false;
+
+    /* Even if we have a flush interval, we may not know the next flush time, so fix that. */
+    if (opts->tiered_flush_next_us == 0 && opts->tiered_flush_interval_us != 0)
+        opts->tiered_flush_next_us = now + opts->tiered_flush_interval_us;
+
     if (do_flush_tier != NULL && opts->tiered_flush_next_us != 0 &&
       opts->tiered_flush_next_us < wake_time) {
         wake_time = opts->tiered_flush_next_us;
```

Reply:
@ddanderson I think we do not need this change as it will never get executed because my understanding is that `opts->tiered_flush_next_us` will never be 0 when we call `testutil_tiered_sleep` since `opts->tiered_flush_next_us` will already be set in `testutil_tiered_flush_complete` which gets called from `testutil_tiered_begin`.

The function `testutil_tiered_begin` is called at the beginning to begin processing tiered storage and initialise a random delay for the first flush.

Reply:
Okay, good enough!